lib/src/steering/kinematics/max_acceleration_kinematics.dart

```diff
@@ -44,4 +45,8 @@ class MaxAccelerationKinematics extends Kinematics {
 
   @override
   Seek seek(Vector2 point) => SeekForMaxAcceleration(owner: own, point: point);
+
+  @override
+  Flee flee(List<Vector2> targets) =>
+      FleeForMaxAcceleration(owner: own, targets: targets);
```

Comment:
Functions covering two lines or more should use `{ }` notation, I thought we had that in the analysis options 🤔